solution.js

```diff
@@ -0,0 +1,40 @@
+const strCompression = (str) => {
+  // removes number from input string
+  const noNumStr = str.replace(/[0-9]/g, '');
+  let output = "";
+
+  for (let i = 0; i < noNumStr.length; i++) {
+    // stores current letter
+    let currentLetter = noNumStr[i];
+    let count = 1;
+
+    // checks if next letter is same as current letter
+    while (noNumStr[i + 1] === currentLetter) {
+      count++;
+      i++;
+    }
+
+    // condition to not display single letter number i.e. 1
+    if (count === 1) {
+      output += currentLetter;
+    } else {
+      output += currentLetter + count;
+    }
+  }
+  return output;
+}
+
+// function to display success or failure message
+const testCase = (input, output) => {
+  if (strCompression(input) === output) {
+    console.log("✅ All Tests Passed ✅");
+  } else {
+    console.log("❌ Test Fail ❌");
+  }
+}
```

Comment:
Nice use of extended characters; it doubles as a check for UTF-8 support 👍 

One way to reduce and be more descriptive:

```suggestion
// function to display success or failure message
const testCase = (input, output) => {
  let inputCompressed = strCompression(input);
  let msg = (inputCompressed === output) ? "✅ All Tests Passed ✅" : "❌ Test Fail ❌";
  console.log(msg);
}
```